storage/storage.go

```diff
@@ -386,34 +461,48 @@ func commaSeparatedStrToTags(str string) []string {
 func (storage DBStorage) getReportUpsertQuery() (string, error) {
```

Comment:
why does this function returns queries operating on different tables for different DB engines?

Reply:
IDK I didn't add this function, but I assume it's because code is clear with this function